Documentation/frequently-asked-questions.md

```diff
@@ -14,20 +14,41 @@ Download and import the Identity and Identity for Edge Network extensions
  
 1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
 
-   ```kotlin
+    ### Kotlin
```

Comment:
here we use h4 Java and h3 kotlin, can we remove the java tag and use h4 for kotlin / groovy?
<img width="768" alt="Screenshot 2024-03-26 at 9 55 47 AM" src="https://github.com/adobe/aepsdk-edgeidentity-android/assets/33132425/f6bbb63c-1156-43aa-b6b7-db4cb279bcaa">